docker/dev.app.Dockerfile

```diff
@@ -1,39 +1,22 @@
-# Go to http://localhost/phpreport to use it (username: admin ; password = admin)
+FROM debian:trixie-slim
 
-FROM ubuntu:jammy
+RUN apt update
+RUN apt -y install php php-cli php-pgsql php-xml php-curl composer
 
-MAINTAINER Juan A. Suarez Romero <jasuarez@igalia.com>
+WORKDIR /app
 
-ENV DEBIAN_FRONTEND noninteractive
+ADD composer.json /app
 
-RUN apt-get update
+RUN composer update
+RUN composer install
+RUN composer dump-autoload -o
 
-RUN apt-get -y install postgresql apache2 php php-pgsql php-xml supervisor make docutils-common uglifyjs git
+ADD . /app
+COPY .env.dev /app/.env
```

Comment:
Actually I forgot to remove this line. Since we are using volumes it doesn't make sense to copy anymore, better to let the user manually handle the `.env` file.